client/src/components/Card/Card.stories.js

```diff
@@ -7,7 +7,11 @@ import people from "../../assets/images/add-friend.png";
 import trends from "../../assets/images/trending-hashtags.png";
 import groups from "../../assets/images/protest.png";
 
-storiesOf("Card", module)
+export default {
+  title: "Cards/Card"
+};
```

Comment:
I don't think we need this. Can you go through and delete these from the components?